run_mode: 1   # 0:复位  1:轨迹模式  2:靠近电机  3:远离电机
target_vel: 5000

lead: 5
reduction_ratio: 35  
encoder_lines_num: 2000   # 4倍频后的编码器线数

# if read cmd_config with XmlRpc::XmlRpcValue, the data will be saved in alphabetic order, that is "left back", "left front", "right back", "right front"
# so use "m0 m1 m2 m3" as the key instead of "left front, right front, left back, right back"
cmd_config:
  m0:  # left front
    dev_ind: 0
    can_ind: 0
    driver_id: 5 
    direction: -1
  m1: # right front
    dev_ind: 0
    can_ind: 0
    driver_id: 6 
    direction: -1
  m2: # left back
    dev_ind: 1
    can_ind: 0
    driver_id: 7 
    direction: -1
  m3: # right back
    dev_ind: 1
    can_ind: 0
    driver_id: 8 
    direction: -1
